test/feed.test.js

```diff
@@ -76,5 +76,12 @@ describe('Fost data class tests', () => {
       expect(persisted.lastModified).toEqual('lastModified');
       expect(persisted.etag).toBe(null);
     });
+
+    test('Feed.isDelayed() should return truthy only after Feed.setDelayed()', async () => {
+      const feed = await Feed.byUrl(data.url);
+      expect(await feed.isDelayed()).toBeFalsy();
```

Comment:
Does this not return a pure Boolean?  Can you not use `toBe(false)` and `toBe(true)` below?  If not, we should fix that.

Reply:
Fixed in https://github.com/Seneca-CDOT/telescope/pull/726/commits/ab0ec50816580a3a5e72c4cb48c9d55afe6b0cfa